### PR TITLE
Domains: Show auto renewal date for domains that have auto renew turned on

### PR DESCRIPTION
**Pull request by eltongo**
#### Changes proposed in this Pull Request

This PR updates the domains status card to show the renewal date instead of the expiration date when auto renew is on.

Screenshot 2020-06-16 at 8 38 11 PM

#### Testing instructions

* For a domain you own, enable auto renew
* Verify that it shows the renewal date instead of the expiration date, like in the screenshot above
* Check that the expiration date is shown instead if you disable auto-renew

### CODE REVIEW

File: client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx [hunk 1]
```diff
@@ -345,6 +345,28 @@ class RegisteredDomainType extends React.Component {
 			'domains/new-status-design/new-options'
 		);
 
+		let expiresText;
```
Comment (klimeryk) [inline client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx:348]: Might be good to just move this whole block to a separate function - then you'll keep `render` function smaller and you'll be able to assign the return value from the new function to a `const` variable 👍

**Comment by fditrapani** [comment]
Thanks for making this update! I found two things:

- There's a case I noticed where we say it's renewing but it can't because the card on file isn't valid until then. In this case, we should show the expiry date: 
image

- It might just be on my end because my auto-renew toggle isn't very responsive but I didn't see the copy update on its own when the toggle switched over.

File: client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx (hunk 1 above)
Comment (eltongo) [reply client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx]: Good idea!

**Comment by eltongo** [comment]
Great catch @fditrapani.

**Comment by fditrapani** [comment]
I'm not sure if you're still working on this but I got this error when I tried to load it up again:

image

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3930531

Thank you @eltongo for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3930531).

**Pull request merged**